utilities/similarity/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM nvidia/cuda:9.0-devel-ubuntu16.04 as cuda_devel
+
+FROM tensorflow/serving:1.11.0-gpu
```

Comment:
I don't think we need TensorFlow Serving for this image, so use `tensorflow/tensorflow:1.14.0-gpu-py3` directly. You can then remove the `FROM` above and the `COPY` and `RUN` below.